docs/guides/debug-qiskit-runtime-jobs.ipynb

```diff
@@ -0,0 +1,619 @@
+{
```

Comment:
<blockquote>These results support mathematical operators and can be used to compute figures of merit</blockquote><p> </p><p>nit - This sentence is too similar to the one above (<em>The simulated results support mathematical operations...</em>) and feel redundant. We can change it to something like "Next, apply mathematical operations to compute the absolute difference.".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/2223/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>